pages/index.html

```diff
@@ -5,8 +5,46 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link rel="stylesheet" href="/stylesheets/style.css">
 </head>
 <body>
-  
+  <header>
+    <h1>shannon's corner</h1>
+    <nav>
+        <ul>
+            <li><a href="about.html">about</a></li>
+            <li><a href="portfolio.html">portfolio</a></li>
+        </ul>
+    </nav>
+  </header>
+  <main>
+    <p>My name is Shannon Bellemore and I am currently a software developer student at Ada Developers Academy.
```

Comment:
I would add some padding around this text. It was hard to read the beginning of it on safari.